packages/devtools_app/lib/src/app.dart

```diff
@@ -542,26 +542,34 @@ class SettingsDialog extends StatelessWidget {
         mainAxisSize: MainAxisSize.min,
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          _buildOption(
+          CheckboxSetting(
             label: const Text('Use a dark theme'),
             listenable: preferences.darkModeTheme,
             toggle: preferences.toggleDarkModeTheme,
+            gaEnabledTag: analytics_constants.darkThemeEnable,
```

Comment:
seems verbose adding a separate tag for enable and disable. Can we just pass in a prefix.

Reply:
done